content/who.njk

```diff
@@ -0,0 +1,54 @@
+---
+layout: base-toc.njk
+title: Digital Society | Who we are
+description: Who we are
+---
+<h3 id="not-for-profit">Our beliefs</h3>
+
+<h4 id="not-for-profit">Not-for-profit</h4>
+
+<p>
+  People chose to work on something they believe in, in circumstances they can shape over pure monetary gain.
+  The primary goal of a for-profit business is to create value for its shareholders.
+  We have seen first hand how this can make it stray from its mission or the environment it provides for its workers.
+  To avoid such a possibility, we incorporated a <strong>not-for-profit</strong>
+  <a href="https://find-and-update.company-information.service.gov.uk/company/SC768012">private company limited by guarantee without share capital</a>.
```

Comment:
**nitpick:** newline? I guess it's very long, but we're otherwise using sentence-per-line I guess.